src/folder/schema.ts

```
@@ -0,0 +1,144 @@
+import { currencyEnum, publisherTypeEnum, userRoleEnum } from "@/models/Schema";
+import { z } from "zod";
+
+// Custom Zod transformer for monetary values
+const monetaryString = () => z.string()
+  .refine((val) => !isNaN(parseFloat(val)) && parseFloat(val) > 0, {
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)